src/main/resources/assets/music_control/lang/en_us.json

```diff
@@ -130,66 +153,7 @@
-    "minecraft:music.overworld.dripstone_caves": "Dripstone Caves",
-    "minecraft:music.overworld.eroded_badlands": "Eroded Badlands",
-    "minecraft:music.overworld.flower_forest": "Flower Forest",
-    "minecraft:music.overworld.forest": "Forest",
-    "minecraft:music.overworld.frozen_ocean": "Frozen Ocean",
-    "minecraft:music.overworld.frozen_peaks": "Frozen Peaks",
-    "minecraft:music.overworld.frozen_river": "Frozen River",
-    "minecraft:music.overworld.grove": "Grove",
-    "minecraft:music.overworld.ice_spikes": "Ice Spikes",
-    "minecraft:music.overworld.jagged_peaks": "Jagged Peaks",
-    "minecraft:music.overworld.jungle": "Jungle",
-    "minecraft:music.overworld.lukewarm_ocean": "Lukewarm Ocean",
-    "minecraft:music.overworld.lush_caves": "Lush Caves",
-    "minecraft:music.overworld.mangrove_swamp": "Mangrove Swamp",
-    "minecraft:music.overworld.meadow": "Meadow",
-    "minecraft:music.overworld.mushroom_fields": "Mushroom Fields",
-    "minecraft:music.overworld.ocean": "Ocean",
-    "minecraft:music.overworld.old_growth_birch_forest": "Old Growth Birch Forest",
-    "minecraft:music.overworld.old_growth_pine_taiga": "Old Growth Pine Taiga",
-    "minecraft:music.overworld.old_growth_spruce_taiga": "Old Growth Spruce Taiga",
-    "minecraft:music.overworld.old_growth_taiga": "Old Growth Taiga",
-    "minecraft:music.overworld.plains": "Plains",
-    "minecraft:music.overworld.river": "River",
-    "minecraft:music.overworld.savanna": "Savanna",
-    "minecraft:music.overworld.savanna_plateau": "Savanna Plateau",
-    "minecraft:music.overworld.snowy_beach": "Snowy Beach",
-    "minecraft:music.overworld.stony_peaks": "Stony Peaks",
-    "minecraft:music.overworld.snowy_plains": "Snowy Plains",
-    "minecraft:music.overworld.snowy_slopes": "Snowy Slopes",
-    "minecraft:music.overworld.snowy_taiga": "Snowy Taiga",
-    "minecraft:music.overworld.sparse_jungle": "Sparse Jungle",
-    "minecraft:music.overworld.stony_shore": "Stony Shore",
-    "minecraft:music.overworld.sunflower_plains": "Sunflower Plains",
-    "minecraft:music.overworld.swamp": "Swamp",
-    "minecraft:music.overworld.taiga": "Taiga",
-    "minecraft:music.overworld.warm_ocean": "Warm Ocean",
-    "minecraft:music.overworld.windswept_forest": "Windswept Forest",
-    "minecraft:music.overworld.windswept_gravelly_hills": "Windswept Gravelly Hills",
-    "minecraft:music.overworld.windswept_hills": "Windswept Hills",
-    "minecraft:music.overworld.windswept_savanna": "Windswept Savanna",
-    "minecraft:music.overworld.wooded_badlands": "Wooded Badlands",
-
-    "minecraft:music.nether.basalt_deltas": "Basalt Deltas",
-    "minecraft:music.nether.crimson_forest": "Crimson Forest",
-    "minecraft:music.nether.nether_wastes": "Nether Wastes",
-    "minecraft:music.nether.soul_sand_valley": "Soul Sand Valley",
-    "minecraft:music.nether.warped_forest": "Warped Forest",
+    "minecraft:records/relic": "Aaron Cherof - Relic",
```

Comment:
We might be able to get these names from the disc itself, I can't remember why I didn't do that. Anyway, I'll do the change after the PR if needed